README.md

```diff
@@ -60,6 +60,8 @@ All event types can be found in the [com.sonymobile.tools.gerrit.gerritevents.dt
     * `java-1.6`
         * `maven-3.0.4`
 
+* Java 8 & 9: Works.
```

Comment:
Btw how do you know if you didn't change the compiler setting in the pom?

Reply:
Hi, this change allow us to run tests on Java 9 runtime. Code is still targeting Java 6 (enforced in POM) but 'mvn tests' can be executed on Java 9.

Without this changes tests are working only with a Java 8 runtime. Tested on macos with:
/Library/Java/JavaVirtualMachines/jdk1.8.0_162.jdk
/Library/Java/JavaVirtualMachines/jdk-9.0.1.jdk/

Reply:
I updated the pom to compile with Java 8 after the merge :)